test/spec/controllers/eventDetails.js

```diff
@@ -1,65 +0,0 @@
-'use strict';
```

Comment:
I was having issues with Intellij and debugging. instead of using the actual eventDetails.js file when setting breakpoints it was the test/spec/controllers/eventDetails.js file.  So if I made a code change and then refreshed browser the breakpoints would not be accurate.  So I was forced to stop, save, restart every time I made a change which took longer. If there is a better approach please let me know.